web/src/pages/404.tsx

```diff
@@ -0,0 +1,34 @@
+import { Content } from "@/components/Content";
+import { Button } from "@/components/njwds-extended/Button";
+import { SingleColumnContainer } from "@/components/njwds/SingleColumnContainer";
+import { PageSkeleton } from "@/components/PageSkeleton";
+import { useConfig } from "@/lib/data-hooks/useConfig";
+import { ReactElement } from "react";
+
+const PageNotFound = (): ReactElement => {
+  const { Config } = useConfig();
+
+  return (
+    <PageSkeleton>
+      <main className="usa-section padding-top-0 desktop:padding-top-8" id="main">
+        <SingleColumnContainer>
+          <div className="flex flex-column flex-align-center">
+            <img src="./img/page-not-found-icon.svg" alt="404 icon" />
+            <h2 className="margin-top-4 text-accent-cool-darker">{Config.pageNotFoundError.errorHeader}</h2>
```

Comment:
For accessibility purposes, every page needs to have an `h1` element ([more info](https://fae.disability.illinois.edu/rulesets/HEADING_1/))

You should use an h1 html tag here, but style it to look like an h2 if that's the desired font size/style/etc. Take a look at the `Header` component, for example, to see an h1 styled as an h2